client.go

```diff
@@ -317,6 +316,16 @@ func WithEmitter(e Emitter) Option {
 	}
 }
 
+// Repositories accessor
+func (c *Client) Repositories() *Repositories {
+	return c.repositories
+}
+
+// Templates accessor
+func (c *Client) Templates() *Templates {
+	return c.templates
+}
```

Comment:
Placing these behind an accessors allows them to be enumerated in an interface and tested.